server/utils/attachDocumentsUtils.test.ts

```diff
@@ -112,6 +112,20 @@ describe('attachDocumentsUtils', () => {
       })
     })
 
+    it('uses the typeDescription if the document description is null', () => {
+      const document = documentFactory.build({ id: '123', description: null })
+      const selectedDocument = documentFactory.build({
+        id: '123',
+        description: null,
```

Comment:
I'm not sure what we will recieve from the API if the description is omitted but I can't imagine it will be a JS `null`. 
I think it's safer here to set it as `undefined`. This will cause `documentWithDescription` to break but it can be resurrected using optional chaining like so: `selectedDocument?.typeDescription`. 
The functionality will work the same if we *do* get a null from the API but we're also covered if the property isn't defined.

Reply:
thanks Rich -- interesting!

I've updated the commit.